src/garage/tf/algos/npo.py

```diff
@@ -35,7 +35,8 @@ class NPO(RLAlgorithm):
             Must be specified if running multiple algorithms
             simultaneously, each using different environments
             and policies.
-        max_episode_length (int): Maximum length of a single episode.
+        max_episode_length (int): Maximum length of a single episode. Should
```

Comment:
i think this can always be calculated as `env.spec.max_episode_length * episodes_per_trial`

Reply:
Yes, `RL2` as an algorithm should have `env.spec.max_episode_length` as `max_episode_length`.
The inner algorithm of `RL2` (e.g. `RL2PPO`) concatenates all paths across `n` episodes and processes as one single giant path, so it should have `env.spec.max_episode_length * episodes_per_trial` as `max_episode_length`.

Reply:
To run the inner algorithm with `max_episode_length= episodes_per_trial * env.spec.max_episode_length` we have to pass it either `episodes_per_trial` or `max_episode_length`. I do the latter here because I think it makes more sense since episodes_per_trial is RL2 specific.